examples/egui/src/main.rs

```diff
@@ -6,8 +6,8 @@ struct EguiExampleData {
     _directional_handle: rend3::types::DirectionalLightHandle,
 
     egui_routine: rend3_egui::EguiRenderRoutine,
-    platform: egui_winit_platform::Platform,
-    start_time: instant::Instant,
+    egui_context: egui::Context,
+    winit_state: egui_winit::State,
```

Comment:
The egui context is now kept outside of the winit integration, so we manage it in the example code.